source/docs/software/advanced-controls/filters/slew-rate-limiter.rst

```diff
@@ -82,7 +82,7 @@ A typical use of a SlewRateLimiter is to limit the acceleration of a robot's dri
     drivetrain.ArcadeDrive(forward, turn);
 
     // Slew-rate limits the forward/backward input, limiting forward/backward acceleration
-    drivetrain.ArcadeDrive(filter.Calculate(forward), turn);
+    drivetrain.ArcadeDrive(filter.Calculate(units::voltage::volt_t{forward}).value(), turn);
```

Comment:
This still hasn't been fixed wrapping something in a constructor only to get the underlying double HAS to be wrong